hazelcast-client/src/main/resources/hazelcast-client-full-example.yaml

```diff
@@ -588,7 +585,6 @@ hazelcast-client:
       max-idle-seconds: 100
       eviction-policy: LFU
       invalidate-on-change: true
```

Comment:
You can remove the `max-size: 2000` and `eviction-policy: LFU` here.

Reply:
done.